src/tools/cleaner.js

```diff
@@ -3,9 +3,9 @@
 const _ = require('lodash');
 const { defaults, pick, pipe } = require('lodash/fp');
 
-const isPlainObj = require('./data').isPlainObj;
-const recurseReplace = require('./data').recurseReplace;
-const flattenPaths = require('./data').flattenPaths;
+const { isPlainObj, getObjectType } = require('./data');
+const { recurseReplace } = require('./data');
+const { flattenPaths } = require('./data');
```

Comment:
Nice cleanup! Can we make it even better by importing them in one statement?

```js
const { isPlainObj, getObjectType, recurseReplace, flattenPaths } = require('./data');
```

Reply:
Haha yikes. I just now noticed that those were coming from the same file. Thanks for that!